frontend/src/app/modules/header/header-item/header-item.component.ts

```diff
@@ -30,10 +29,12 @@ export class HeaderItemComponent extends BaseComponent implements OnInit {
     }
 
     ngOnInit(): void {
+        this.currentUserImage = this.userService.getLocalUser()?.image;
+
```

Comment:
Why you cannot use the event?

Reply:
because when the page is reloaded and no user is needed on it, the event has the value undefined

Reply:
looks like workaround and not a solution

Reply:
Fixed